platform/linux-generic/odp_packet.c

```diff
@@ -1588,6 +1598,9 @@ int odp_packet_is_valid(odp_packet_t pkt)
 	if (odp_event_type(ev) != ODP_EVENT_PACKET)
 		return 0;
 
+	if (_odp_packet_validate(pkt, _ODP_EV_PACKET_IS_VALID))
```

Comment:
Unlikely? Also with `_odp_buffer_validate()` in `odp_buffer_is_valid()`.

Reply:
Both changed in V4.